docs/src/usage.md

```diff
@@ -28,8 +28,6 @@ Common to both `kani` and `cargo kani` are many command-line flags:
  If used with `print`, Kani will only print the unit test to stdout.
  If used with `inplace`, Kani will automatically add the unit test to the user's source code, next to the proof harness. For more detailed instructions, see the [debugging verification failures](./debugging-verification-failures.md) section.
 
- * `--visualize`: Generates an HTML report showing coverage information and providing traces (i.e., counterexamples) for each failure found by Kani.
```

Comment:
I think it would be helpful to leave this in the documentation, but with the caveat that it's not stable yet. Would that work?

Reply:
Yes, just now I've restored it and added the text we use in concrete playback to highlight it's experimental (i.e., ``_Experimental_, `--enable-unstable` feature that ...``).